src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -9520,34 +9520,34 @@ public void M()
     {
         Out(out string s2);
         s2 /*T:string!*/ .ToString();
-        s2 = null; // warn 1
+        s2 = null; // 1
 
         NOut(out string? ns2);
-        ns2 /*T:string?*/ .ToString(); // warn 2
+        ns2 /*T:string?*/ .ToString(); // 2
         ns2 = null;
 
         FalseOut(out string s3);
         s3 /*T:string!*/ .ToString();
-        s3 = null; // warn 3
+        s3 = null; // 3
 
         FalseNOut(out string? ns3);
-        ns3 /*T:string?*/ .ToString(); // warn 4
+        ns3 /*T:string?*/ .ToString(); // 4
         ns3 = null;
 
         Out(out var s4);
         s4 /*T:string!*/ .ToString();
-        s4 = null; // warn 5
+        s4 = null; // 5
 
         NOut(out var ns4);
```

Comment:
I expected that before this PR, this was a warning. Did I miss something in Friday's discussion? No big deal either way.

Reply:
Specifically, the location `out var ns4` would contain a warning about a null assignment.

Reply:
(This had to do with combination with [MaybeNull], [MaybeNullWhen], so I resolved my confusion. Ignore these comments)